GoMeet/src/main/java/com/google/sps/servlets/LocationServlet.java

```diff
@@ -0,0 +1,46 @@
+package main.java.com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import main.java.com.google.sps.data.Location;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+/** Handles fetching and saving location data. */
+@WebServlet("/location-data")
+public class LocationServlet extends HttpServlet {
+  /** Accepts a POST request containing a new location. */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) {
+    double lat = Double.parseDouble(request.getParameter("lat"));
```

Comment:
Will this crash if there was no 'lat' and 'lng' parameter in the request? 
Or if the values in the 'lat' and 'lng' params were not numbers?

Reply:
(Answered in another comment)